src/BenchmarkDotNet/Jobs/EnvironmentVariable.cs

```diff
@@ -1,17 +1,43 @@
-﻿namespace BenchmarkDotNet.Jobs
+﻿using System;
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Jobs
 {
     public class EnvironmentVariable
```

Comment:
missing `IEquatable` interface implementation 

```suggestion
    public class EnvironmentVariable : IEquatable<EnvironmentVariable>
```